.github/workflows/release-python.yml

```diff
@@ -0,0 +1,89 @@
+name: Python Release
+
+on:
+  push:
+    tags:
+      - "pymongocrypt-[0-9]+.[0-9]+.[0-9]+"
+      - "pymongocrypt-[0-9]+.[0-9]+.[0-9]+.post[0-9]+"
+      - "pymongocrypt-[0-9]+.[0-9]+.[0-9]+[a-b][0-9]+"
+      - "pymongocrypt-[0-9]+.[0-9]+.[0-9]+rc[0-9]+"
+  workflow_dispatch:
+
+concurrency:
+  group: wheels-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: ./bindings/python
+    shell: bash -eux {0}
+
+jobs:
+  build_dist:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-latest, macos-latest]
```

Comment:
Why use windows-latest and macos-latest vs ubuntu-20.04? Should we pin the windows/mac hosts too?

Reply:
Works with ubuntu-latest